Automattic/newspack-blocks
# fix: resolve merge conflict with release

### All Submissions:






### Changes proposed in this Pull Request:



The post-release job after the latest hotfix release is experiencing failures due to a merge conflict. This should fix the conflict.

### How to test the changes in this Pull Request:

Verify that `npm ci --legacy-peer-deps && npm run build` run successfully, and that all CI jobs succeed below. 👇 

### Other information:
